rotkehlchen/chain/evm/decoding/airdrops.py

```diff
@@ -0,0 +1,57 @@
+from typing import TYPE_CHECKING, Optional
+
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.chain.ethereum.modules.airdrops.constants import CPT_COW
+from rotkehlchen.chain.ethereum.utils import asset_normalized_value
+from rotkehlchen.chain.evm.decoding.structures import (
+    DEFAULT_DECODING_OUTPUT,
+    DecoderContext,
+    DecodingOutput,
+)
+from rotkehlchen.utils.misc import hex_or_bytes_to_address, hex_or_bytes_to_int
+
+if TYPE_CHECKING:
+    from rotkehlchen.accounting.structures.evm_event import EvmEvent
+    from rotkehlchen.assets.asset import Asset, CryptoAsset
+    from rotkehlchen.fval import FVal
+    from rotkehlchen.types import ChecksumEvmAddress
+
+COW_PROTOCOL_CLAIMED = b'\xd46\xe9\x97=\x1eD\xd4\r\xb4\xd4\x11\x9e<w<\xad\xb12;&9\x81\x96\x8c\x14\xd3\xd1\x91\xc0\xe1H'  # noqa: E501
+
+
+def make_claim(
+        event: 'EvmEvent',
+        user_address: 'ChecksumEvmAddress',
+        amount: 'FVal',
+        asset: 'Asset',
+        counterparty: str,
+        notes: Optional[str] = None,
+) -> bool:
+    """It matches an transfer event to an airdrop claim, changes the required fields
+     then returns `True` if a match was found"""
+    if not (event.event_type == HistoryEventType.RECEIVE and event.location_label == user_address and amount == event.balance.amount and asset == event.asset):  # noqa: E501
+        return False
+    event.event_subtype = HistoryEventSubType.AIRDROP
+    event.counterparty = counterparty
+    event.notes = f'Claim {amount} {asset.symbol_or_name()} from {counterparty} airdrop' if notes is None else notes  # noqa: E501
+    return True
+
+
+def decode_cow_claim(context: DecoderContext, vcow: 'CryptoAsset') -> DecodingOutput:
```

Comment:
What is this doing here? This is a very specific airdrop claim located in a file that should only contain generic code.

This file is in `/evm/decoding/airdrops.py` and should only have generic code.

Also why is a constant asset passed as an argument? :thinking: 

This should just be part of the decoder and use the `self.vcow`  ... that's not a place for it to be.